src/ert/parsing/lark_parser.py

```diff
@@ -190,10 +189,10 @@ def _tree_to_dict(
     pre_defines: Defines,
     tree: Tree[Instruction],
     site_config: Optional[Dict[str, Any]] = None,
+    schema: Optional[Mapping[str, SchemaItem]] = None,
 ) -> Mapping[str, Instruction]:
-    schema: Mapping[str, SchemaItem] = (
-        init_user_config() if site_config is not None else init_site_config()
-    )
+    if schema is None:
+        schema = init_user_config() if site_config is not None else init_site_config()
```

Comment:
I think its probably simpler to just make this a required parameter and insert it into those two places where it was not present before.